msteams-platform/get-started/deploy-csharp-app-studio.md

```diff
@@ -19,33 +19,33 @@ Complete the following steps to update the app package:
 
 1. To install App Studio in Teams, select the **Apps** icon at the bottom of the left-hand bar, and search for **App Studio**:
 
-    <img  width="450px" alt="Finding App Studio in the Store View" src="~/assets/images/get-started/searchforAppStudio.png"/>
+    :::image type="content" source="~/assets/images/get-started/searchforAppStudio.png" alt-text="Finding app studio in the store view.":::
 
 1. Select the **App Studio** tile and choose **Install**. The App Studio is installed:
 
-    <img  width="450px" alt="Installing App Studio" src="~/assets/images/get-started/InstallingAppStudio.png"/>
+    :::image type="content" source="~/assets/images/get-started/InstallingAppStudio.png" alt-text="Installing app studio.":::
 
 1. To create the app package for your Teams app, select the **Manifest editor** tab in **App Studio**:
 
-    <img  width="450px" alt="App Studio" src="~/assets/images/get-started/AppStudio.png"/>
+    :::image type="content" source="~/assets/images/get-started/AppStudio.png" alt-text="The screenshot describes on how to select manifest editor to create app package.":::
```

Comment:
"The screenshot describes on" reads weird. There are many instances of this usage in the PR. 

Kindly address the copy issues with the help of an internal review or with an editor's help. As many copy issues in a PR are not done. @v-habibaumme , @surbhigupta12

Reply:
changed the "describes on" to "shows the steps to"